app_template_dev.py

```diff
@@ -52,20 +55,39 @@
 region = os.environ["CDK_DEFAULT_REGION"]
 account = os.environ["CDK_DEFAULT_ACCOUNT"]
 
+# Should be the account number for either dev or prod, depending
+# on where the backups are coming from. Optional parameter
+try:
+    s3_source_account = os.environ["CDK_S3_BACKUPS_SOURCE_ACCOUNT"]
+except KeyError:
+    pass
+
 env = Environment(account=account, region=region)
 app = App()
-params = app.node.try_get_context("dev")
+params = app.node.try_get_context(CONTEXT)
 # sds_id = "abc-dev"
 sds_id = f"{INITIALS}-{params['sds_id']}"
 
 print(f"Using the profile [{current_profile}] in region [{region}].")
 print(f"The stack identifier being used is: {sds_id}")
 
-stacks = build_sds(
-    app,
-    env=env,
-    sds_id=sds_id,
-    use_custom_domain=True,
-)
+if params["sds_id"] == "backup":
+    if not s3_source_account:
+        raise KeyError(
+            "No source account is set for the backup deploy."
+            "Please define the CDK_SOURCE_ACCOUNT environment variable."
```

Comment:
should this be `CDK_S3_BACKUPS_SOURCE_ACCOUNT` instead?

Reply:
Yes! Thanks for the catch, I changed the name